### ISSUE TITLE
Companion error with Propresenter 

### ISSUE THREAD

**Issue by TheChapel24** [bitfocus/companion-module-renewedvision-propresenter#197]
When I connect companion to ProPresenter fro some reason it keeps saying "timed out error" under configuration.

**Comment by dfountainLBC** [comment]
I am having the same issue, would love to know if it resolves

**Comment by thebeardwon1** [comment]
Seeing the same issue. Two separate companion servers. 1 server that controls one Pro7 computer and 1 server that reaches two different Pro7 computers.

**Comment by thebeardwon1** [comment]
**FIX:** I was able to get around the issue by changing "Type of Presentation Info Requests" to Manual, instead of Automatic. It's 2/3 of the way down, under "Workaround Settings." 

_NOTE: I did have to close ProPresenter and restart companion to be able to get out of the connecting loop to be able to access the "Edit Connections" settings._

**Comment by TheChapel24** [comment]
is the "type of presentation info requests" to manual within companion? were can I do this? can't seem to find this

**Comment by thebeardwon1** [comment]
Yes. Its in the module settings in the Connections tab of Companion. Screenshot attached. If you can't see the settings because you are still stuck in the "connecting loop," close ProPresenter, restart your Companion server and then you can access the settings. If ProPresenter is open, it will get back into the loop. 

![Screenshot 2024-08-21 at 11 23 19 AM](https://github.com/user-attachments/assets/1f3e255f-b792-46d4-af57-e801850170a9)

**Comment by TheChapel24** [comment]
Thank you! it is now working! Hopefully it holds up!

**Comment by thebeardwon1** [comment]
It worked great for us on both machines on our server all weekend.

**Comment by TheChapel24** [comment]
by any chance are you triggering hot keys in your set up? I would like to have hot keys like bridge & chorus commands for our volunteers

**Comment by thebeardwon1** [comment]
We don't. We actually don't have an operator on lyrics anymore. It is all Midi triggered by our tracks. 

I don't see anything for triggering hot keys, but I think you could achieve the same outcome by using the "Specific Slide in a Group" function and just setting it to Slide 1 of [group name]. If you don't put anything in the Presentation Path box, it will just trigger the first slide of that group in the current presentation/song. 

![Screenshot 2024-08-21 at 11 41 07 AM](https://github.com/user-attachments/assets/582dfec6-8c7f-4e81-bc8d-1317f2489c7d)

**Comment by TheChapel24** [comment]
Thank You! would something like this work if I wanted to create a button to trigger an specific song within our playlist?

**Comment by thebeardwon1** [comment]
Yes. Then you _would_ use the Presentation Path field. You put [playlist number]:[presentation number] into that field. One note there, the first playlist = 0 and the first presentation = 0.

So let's say your song is in the first playlist and is the second item, maybe behind an announcement loop. You would enter 0:1 into the Presentation Path. 0=first playlist, 1=second presentation in that playlist. 

That can get confusing, even more so because Slide Number uses the actual number on the slide and does not do the 0=1 thing.
